### PR TITLE
slack connector

### PR DESCRIPTION
**Pull request by gregory28seo**

### CODE REVIEW

**Comment by diminishedprime** [comment]
Thanks for the pull request, code looks pretty good. Would you mind switching over the schema and config to use the new `DataStudioApp` builders? You can see what this looks like on PR#83.

**Comment by gregory28seo** [comment]
> e pull request, code looks pretty good. Would you mind switching over the schema and config to use the new `DataStudioApp` builders? You can see what this looks like on PR#83.

Thanks a lot for your suggestions, I have made the necessary changes.

File: slack/src/code.gs [hunk 1]
```diff
@@ -115,13 +149,18 @@ function getSchema(request) {
...
+  });
+  var requestedFields = getFields().forIds(requestedFieldIds);
+  
   var url = [
     'https://slack.com/api/channels.history?token=',
     request.configParams.token,
     '&channel=',
     request.configParams.channel,
     '&count=',
-    request.configParams.count,
```
Comment (diminishedprime) [inline slack/src/code.gs]: Shouldn't this still use the configParam?

Comment (gregory28seo) [reply slack/src/code.gs]: That's done

**Pull request merged**